src/components/toolbar/ToolBar.vue

```diff
@@ -8,11 +8,16 @@
         <TopRail
           :validation-errors="validationErrors"
           :warnings="warnings"
-        />
+        >
+          <component
+            :is="component.button"
+            v-for="(component, index) in validationBar"
+            :key="`validation-status-${index}`"
+          />
+        </TopRail>
 
-        <align-buttons @save-state="$emit('save-state')" />
-
-        <div class="btn-group btn-group-sm mr-2" role="group" aria-label="Additional controls">
+        <!-- TODO Remove this block when redesigning -->
```

Comment:
As the feature is complete, it looks that this commented code can be removed

Reply:
It will be removed